api/libproject/devstart.rb

```diff
@@ -1524,6 +1524,60 @@ def load_institutions(cmd_name, *args)
     :fn => ->(*args) {load_institutions(LOAD_INSTITUTIONS_CMD, *args)}
 })
 
+def populate_ops_user_affiliations(cmd_name, *args)
```

Comment:
This is all copy, paste, substitute.  So I'm warning you ahead of time: I'm probably going to dismiss most comments here with "not today, but keep that in mind for the big rewrite!"